"0","filtered_df
"
"2","Error in gzfile(file, ""wb"") : bağlantı açılamadı
"
